templates/components/vectordbs/python/none/vectordb.py

```diff
@@ -8,6 +8,8 @@ def get_vector_store():
     if not os.path.exists(STORAGE_DIR):
         vector_store = SimpleVectorStore()
     else:
-        vector_store = SimpleVectorStore.from_persist_dir(STORAGE_DIR)
+        vector_store = SimpleVectorStore.from_persist_dir(
+            STORAGE_DIR, namespace="default"
```

Comment:
1. isn't `default` the default namespace - do we need to specify it?
2. can't we just use:
`vector_store = SimpleVectorStore.from_persist_dir(
            STORAGE_DIR)` also if the path `STORAG_DIR` doesn't exit?

Reply:
Oh, I see. It can be a bit confusing. Basically, the llama_index stores the index with a prefix (namespace). However, when loading it, if we don't specify the namespace, it won't use the default one.

Great catch! we should add a logic to assert the missing STORAGE_DIR if it doesn't exit.

Reply:
About 2. I mean why do we need the case with SimpleVectorStore()

Reply:
Ah, understand. The SimpleVectorStore() case is for generating step where it initializes a new simple vector store instance. Once we already have the vector store (persisted previously) then we should initialize the vector store with persisted data.

Reply:
got it i'll add some comments